_posts/2016-07-28-community-testing-day-on-20160805.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Community Testing Day on August 5th
+layout: post
+author: Chongrui Zhang
+---
+
+The Pulp 2.10 GA is [scheduled for Aug 16, 2016](https://pulp.plan.io/projects/pulp/wiki/Release_Schedule). To deliver a robust Pulp release, the Pulp QE and Pulp team have organized a Community Test day where the Pulp community can come together to test Pulp. This post further details the event and how to participate it.
+
+# What is a Community Testing Day and why host it?
+
+According to the Fedora blog, organizing a test day "is a great way to help expose your project and bring more testers to trialing a new update before it goes live".
+
+The main objective of hosting this Pulp Test Day is to identify as many issues as possible early on with the help of the community, get them fixed as quickly as possible and thus have a more stable release.
+
+# What to test?
+
+The focus of the test day will be the new features included in Pulp 2.10. The issues fixed in 2.10 can also be verified that day.  A list of these features will be provided, and participants can sign up for the features they want to test. The attendees are welcomed to test existing features as well. Members can also participate in the test day by adding tests to our automation test suite (Pulp Smash) or by submitting fixes to the reported bugs. All issues and bugs will be filed and triaged to prepare for 2.10 GA.
```

Comment:
Every period should have only 1 space after it. Some of these have 2. Similar throughout the article.